case_utils/case_validate/__init__.py

```diff
@@ -255,14 +353,13 @@ def main() -> None:
     # determination by output file extension.  case_validate will defer
     # to pySHACL behavior, as other CASE tools don't (at the time of
     # this writing) have the value "human" as an output format.
-    validator_kwargs: typing.Dict[str, str] = dict()
+    validator_kwargs: Dict[str, str] = dict()
     if args.format != "human":
         validator_kwargs["serialize_report_graph"] = args.format
 
-    validate_result: typing.Tuple[
-        bool, typing.Union[Exception, bytes, str, rdflib.Graph], str
-    ]
-    validate_result = pyshacl.validate(
+    validate_result: Tuple[
+        bool, Union[Exception, bytes, str, rdflib.Graph], str
```

Comment:
Also, this patch-state includes `from rdflib import Graph`; I'm surprised `rdflib.Graph` here didn't cause `mypy` to balk.

Reply:
Ah, nevermind, the `import rdflib` line wasn't removed.  That's why `rdflib.Graph` still works.